packages/konsol/src/user-add.js

```diff
@@ -0,0 +1,114 @@
+const config = require("@todastic/config");
+const { initDatabase } = require("@todastic/server-web/src/server/lib/database-mongo.js");
+const { createUserModel } = require("@todastic/storage-users");
+const { createEventModel } = require("@todastic/storage-events");
+const parseArgs = require("minimist");
+
+const args = parseArgs(process.argv.slice(2))
+
+if (args.h) {
+  console.log("USAGE: user-add -u <username> -p <password> -l <en|de>");
+  console.log("Password and language are optional.");
+  console.log("If you don't provide one, a password will be generated for you and displayed on the console.");
+  console.log(" If you don't provide a language, we assume english.");
+  return 0;
+}
+
+createUser(args).then(
+  c => console.log(c),
+  err => {
+     console.log("Error!", err);
+     process.exit(2);
+    });
+
+async function createUser(args) {
+  const username = args.u;
+  const password = args.p || generatePassword();
+  const language = args.l || defaultLanguage();
+  
+  if (username === undefined) {
+    throw "Need username";
+  }
+
+  const db = await initDatabase({ config, logger: console })
+  const User = createUserModel({ mongoose: db.mongoose });
+  const Event = createEventModel({ mongoose: db.mongoose });
+  const user = await User.create({ username, password })
+  const uid = user._id + "-id-";
+
+  let events;
+  if (language === "de") {
+    events = eventsDe(user, uid);
+  } else if (language === "en") {
+    events = eventsEn(user, uid);
+  } else {
+    throw "Don't know what to do with language entry '" + language + "'";
+  }
+  await events.map(event => {
+    e = Event.create(event)
+    process.stdout.write(".")
+    return
+  });
+  console.log("");
+
+  await db.mongoose.disconnect();
+  return "All done. User " + username + " with password " + password + " successfully created.";
+}
+
+function generatePassword() {
+  console.log("No password given, generating one.");
+  password = Math.random().toString(36).slice(-8);
```

Comment:
Why `.slice(-8)`?

Reply:
No real reason. Thought it might become too long maybe. But doesn't seem to. Will remove the slice.